cdk/lib/rest-endpoints.ts

```diff
@@ -47,7 +47,7 @@ export class RestEndpoints extends Construct {
       fileName: "rest-endpoints.zip",
       functionName: `recipes-backend-rest-endpoints-${scope.stage}`,
       handler: "main.handler",
-      memorySize: 128,
+      memorySize: 256,
```

Comment:
Interesting. Any metrics or observations suggesting this was struggling with less memory?

Reply:
Yeah the logs show the total amount of used RAM, and I saw it was up fairly close to 128. Also, bumping RAM gives a higher CPU cap => faster execution